client/optimizing.go

```diff
@@ -283,9 +283,6 @@ func parallelGet(ctx context.Context, clients []Client, round uint64, timeout ti
 					rr := get(gctx, c, round)
 					cancel()
 					if rr != nil {
-						if rr.result != nil && rr.result.Round() != round {
```

Comment:
if `round != 0` this still seems very much worth at least an info-level log?

Reply:
Not here though. Maybe the verifier should be logging that?